platform/darwin/src/MGLMapSnapshotter.h

```diff
@@ -12,40 +12,41 @@ MGL_EXPORT
 @interface MGLMapSnapshotOptions : NSObject
 
 /**
- Creates a set of options with the minimum required information
+ Creates a set of options with the minimum required information.
+ 
  @param styleURL the style url to use
  @param camera the camera settings
  @param size the image size
  */
 - (instancetype)initWithStyleURL:(NSURL*)styleURL camera:(MGLMapCamera*)camera size:(CGSize)size;
 
-#pragma mark - Configuring the map
+#pragma mark - Configuring the Map
 
 /**
   The style URL for these options.
  */
-@property (nonatomic, readonly) NSURL* styleURL;
+@property (nonatomic, readonly) NSURL *styleURL;
 
 /**
- The zoom. Default is 0.
+ The zoom level. Default is 0.
  */
-@property (nonatomic) double zoom;
+@property (nonatomic) double zoomLevel;
 
 /**
  The `MGLMapcamera` options to use.
```

Comment:
Like the `coordinateBounds` documentation, this comment should explain what happens if both it and `coordinateBounds` are set.

What happens if the camera specifies an altitude that differs from the `zoomLevel` property? What value ends up being preferred?